aspnet/Grid/How-to/Custom-Binding-for-Grid-CRUD-operation.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Custom Binding for Grid CRUD operation | Grid | ASP.NET Webforms | Syncfusion
-description: custom binding for grid crud operation
+title: Custom binding in ASP.NET Web Forms Grid Control | Syncfusion
+description: Learn here all about custom binding for grid crud operation in Syncfusion ASP.NET Web Forms Grid control, its elements and more.
 platform: aspnet
 control: Control Name undefined
 documentation: ug
 ---
 
-### Custom Binding for Grid CRUD operation
+# Custom Binding for Grid CRUD operation
```

Comment:
Is this a proper description? where is the platform and control names?
![image](https://github.com/syncfusion-content/asp.netwebforms-docs/assets/90315799/6efd4f5f-9b48-4341-b0a8-3207d873b67f)

Reply:
changed